config/components/coverage/manager_e2e_coverage_patch.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
-  namespace: system
+  name: operator-controller-controller-manager
+  namespace: olmv1-system
```

Comment:
I think both was changed when we run kustomize build
Could we not leave as before for we centralize the namespace name and prefix in the kustomization.yaml instead?

Reply:
No, this is a patch for a specific deployment. Since the configurations are combined, there are two "controller-manager" deployments before their prefixes are applied. We need to be specific here.

Reply:
Error without these changes:
```
Error: accumulating components: accumulateDirectory: "recursed accumulation of path '/home/tshort/git/operator-framework/operator-controller/config/components/coverage': multiple matches for Id Deployment.v1.apps/controller-manager.system; failed to find unique target for patch Deployment.v1.apps/controller-manager.system"

Reply:
That is fine; we can always change things if we need them in the future as well.
i think we would need a kustomize for each